srcpkgs/Signal-Desktop/template

```diff
@@ -14,14 +14,16 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="AGPL-3.0-only"
 homepage="https://github.com/signalapp/Signal-Desktop"
 distfiles="https://github.com/signalapp/Signal-Desktop/archive/v${version}.tar.gz"
-checksum=fb3e59e853b16a99dee5db556b45dd19694b9c7f5e2505dcb546957c7c9bd26a
+checksum=4b695219fcd347ae41590633a7e4c461bf98bceeb5254b3cf744663ac2703ee0
 nostrip_files="signal-desktop"
 
 post_extract() {
 	# git-lfs hook needs to be installed for one of the dependencies
 	git lfs install
 
-	vsed 's/"node": "/&>=/' -i package.json
+	# FIXME: We currently don't have the latest Node so let's force Signal
+	# to use what we have. This is very far from ideal.
+	vsed 's/"node": ".*"/"node": ">=18.19.1"/' -i package.json
```

Comment:
You should use a real patch for this, not `vsed`.

Reply:
I thought so too but decided to stick to what was already there :/ Let's address this next cycle... I just noticed it was maybe copied from Arch.

Reply:
I will take a stab on Node 20 on the weekend so this won't be for too long I hope.